### ISSUE TITLE
image resize

### ISSUE THREAD

**Issue by ShiAngWang** [yun-liu/DEL#8]
Hi， while training， have you resized the images into a specific size or used the original image without transformation?  If you resized the images, can you show where the code is?

**Comment by yun-liu** [comment]
We just used the original image size without resizing. To do this, we used a batch size of 1 ([code](https://github.com/yun-liu/DEL/blob/master/examples/del/train.prototxt#L21)) and set the `iter_size` in Caffe to 5 ([code](https://github.com/yun-liu/DEL/blob/master/examples/del/solver.prototxt#L6)). In this way, the gradients during training are averaged among 5 images, similar to the effect of the batch size of 5 (we don't use batchnorm, so `iter_size=5` euqals to `batch size=5` here).
